### PR TITLE
Update solar system

### PR DESCRIPTION
**Pull request by YingboMa**
Close https://github.com/SciML/SciMLTutorials.jl/issues/38

### CODE REVIEW

**Comment by ChrisRackauckas** [comment]
Welcome back.

**Comment by ChrisRackauckas** [comment]
@staticfloat could we get some help here? I don't know what's going on.

**Comment by staticfloat** [comment]
I've restricted it to use only `x86_64` agents.  :)

**Comment by ChrisRackauckas** [comment]
Method errors! @yingboma

**Comment by YingboMa** [comment]
How can I check the built tutorials?

**Comment by ChrisRackauckas** [comment]
Open the Buildkite, go down to the job, click the artifacts. You'll see the .md and the images.

**Comment by ChrisRackauckas** [comment]
The long awaited child has returned.

File: README.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ SciMLTutorials.open_notebooks()
   - [Tutorial on using spatial SSAs in DiffEqJump](http://tutorials.sciml.ai/html/jumps/spatial.html)
   - [Kepler Problem Orbit](http://tutorials.sciml.ai/html/models/05-kepler_problem.html)
   - [Spiking Neural Systems](http://tutorials.sciml.ai/html/models/08-spiking_neural_systems.html)
+  - [Outer Solar System](http://tutorials.sciml.ai/html/models/05-outer_solar_system.html)
```
Comment (YingboMa) [inline README.md:55]: This link doesn’t seem to work.

Comment (ChrisRackauckas) [reply README.md:55]: The HTMLs aren't built anymore.

Comment (ChrisRackauckas) [reply README.md:55]: It should build to https://tutorials.sciml.ai/dev/ soon

**Pull request merged**